wgengine/magicsock/magicsock.go

```diff
@@ -578,14 +578,12 @@ func NewConn(opts Options) (*Conn, error) {
 	c.donec = c.connCtx.Done()
 	c.netChecker = &netcheck.Client{
 		Logf:                logger.WithPrefix(c.logf, "netcheck: "),
-		GetSTUNConn4:        func() netcheck.STUNConn { return c.pconn4 },
+		GetSTUNConn4:        func() netcheck.STUNConn { return &c.pconn4 },
 		SkipExternalNetwork: inTest(),
 		PortMapper:          c.portMapper,
 	}
 
-	if c.pconn6 != nil {
-		c.netChecker.GetSTUNConn6 = func() netcheck.STUNConn { return c.pconn6 }
-	}
+	c.netChecker.GetSTUNConn6 = func() netcheck.STUNConn { return &c.pconn6 }
```

Comment:
I thought this might've been sad, making this hook always populated non-nil even when there was no kernel IPv6 support, but it turns out that's actually better behavior! otherwise netcheck tries to bind to ipv6 again just to waste time failing.